### DISCUSSION TITLE
TASK [paperless-ng : install jbig2enc]  FAILED

### DISCUSSION THREAD

**Discussion by Schrello** [jonaswinkler/paperless-ng#1437] [Support]
Hey, 

I am trying to install paperless-ng via ansible and get the following error:

TASK [paperless-ng : install jbig2enc] *******************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "No package matching 'jbig2enc' is available"}

I tried to install the jbig2enc package manuel as recommended here: [https://githubmemory.com/repo/jonaswinkler/paperless-ng/issues/1126](url) using `sudo snap install jbig2enc --edge`

And I tried to install it as reccomended here: [https://ocrmypdf.readthedocs.io/en/latest/jbig2.html](url)

Both installations worked but ansible still fails during the installation. 

Does anyone have a idea?

I am trying to install it on Raspberrypi 3b+ on the RaspberryPi OS

:(

**Comment by siancu** [comment]
You said you tried to install it as recommended here: https://ocrmypdf.readthedocs.io/en/latest/jbig2.html. Did you succeed in following the steps and installing it? Or did it fail at some point? If it failed, then what was the error?

**Reply by Schrello** [reply]
Hey,

the installation as recommended on https://ocrmypdf.readthedocs.io/en/latest/jbig2.html was successfull. But the error while installing paperless-ng still occures.

I changed the OS from Raspbian to Ubuntu and the installation did work. So i guess it has to do something with the OS Raspbian or the Raspberry Pi.
